static/js/components/EducationForm.js

```diff
@@ -126,8 +129,12 @@ class EducationForm extends ProfileFormFields {
     </Cell>;
   }
 
-  renderEducationLevelEntries(level: any): Array<React$Element<*>|void>|void {
+  renderEducationLevelEntries(level: any): Array<React$Element<*>|void>|void|null {
     const { profile } = this.props;
+    if (!profileOfLoggedinUser(profile) && (!profile.education || profile.education.length === 0)) {
+      return null;
+    }
```

Comment:
- we want to stop rendering here for other users if they do not have education or work history.
- But for own profile, if you do not have data i.e education or work history then we will show you empty card with ``Add degree`` or ``Add employment`` button